scripts/gen_profiles_layer.py

```diff
@@ -2791,7 +2791,7 @@ class JsonLoader {
     uint32_t pCount_copy = *pCount;
 
     PhysicalDeviceData *pdd = PhysicalDeviceData::Find(physicalDevice);
-    const uint32_t src_count = pdd ? static_cast<uint32_t>(pdd->simulation_extensions_.size()) : 0;
+    //const uint32_t src_count = pdd ? static_cast<uint32_t>(pdd->simulation_extensions_.size()) : 0;
```

Comment:
Is this because it's dead code?

Reply:
Yes, variable was unused, the code where it was used before is also commented out